vignettes/fec_vignette.Rmd

```diff
@@ -47,36 +44,64 @@ We used the tame data principles to produce uniform data frames that are easy to
 4. Missing data has value `NA`
 5. Removed signs such as % and $ attached to numeric values
 
+## Who should use this package? 
+
+Anyone interested in US politics and elections who wants to use actual data to think critically and make inferences. We made this package particularly with students and instructors in mind therefore this collection of data is good for teaching. 
+
```

Comment:
Nice. I would add something about relational data, since there is demand for teaching data of this structure.